concent-builder/configure-jenkins.yml

```diff
@@ -0,0 +1,184 @@
+            - { url: "{{ deployment_values_repository }}", name: concent-deployment-values }
+
+        - name: Create directory for concent jenkins secrets
+          file:
+            path:  "{{ jenkins_home_dir }}/{{ item }}"
+            state: directory
+            owner: jenkins
+            group: jenkins
+          with_items:
+            - concent-secrets
+            - concent-secrets/{{ cluster }}/
+            - concent-secrets/cloud/
+            - concent-secrets/jenkins/
+
+    - become:      yes
+      become_user: root
+      block:
+        - name: Upload secrets
+          copy:
+            src:   "{{ local_secret_dir }}/{{ cluster }}/{{ item }}"
+            dest:  "{{ jenkins_home_dir }}/concent-secrets/{{ cluster }}/{{ item }}"
+            owner: jenkins
+            group: jenkins
+          with_items:
+            - cluster-secrets.yml
+            - "../cloud/{{ gke.service_account_name }}-private-key.json"
+            - nginx-proxy-ssl.crt
+            - nginx-proxy-ssl.key
+            - nginx-storage-ssl.crt
+            - nginx-storage-ssl.key
+            - "../jenkins/.token"
+
+        - name: Delete default user settings
+          file:
+            path:  /var/lib/jenkins/users/users.xml
+            state: absent
+
+        - name: Create directory for jenkins job and workspace
+          file:
+            path:  "{{ item }}"
+            state: directory
+            owner: jenkins
+            group: jenkins
+          with_items:
+            - "/var/lib/jenkins/jobs/Build-deploy-and-test/"
+            - "/var/lib/jenkins/jobs/Build-deploy-and-test/workspace"
+            - "/var/lib/jenkins/users/admin/"
+            - "/var/lib/jenkins/secrets/"
+
+        - name: Upload jenkins secrets
+          copy:
+            src:   "{{ local_secret_dir }}/jenkins/{{ item.name }}"
+            dest:  "/var/lib/jenkins{{ item.path }}{{ item.name }}"
+            owner: "jenkins"
+            group: "jenkins"
+          with_items:
+            - { path: "/users/admin/", name: "config.xml" }
+            - { path: "/users/admin/", name: "apiTokenStats.xml" }
+            - { path: "/", name: "credentials.xml" }
+            - { path: "/secrets/", name: "master.key" }
+            - { path: "/secrets/", name: "hudson.util.Secret" }
+
+        - name: Upload jenkins configuration file
+          # FIXME This need use as remote_src
+          copy:
+            src:   "jenkins-configuration-files/{{ item.name }}"
+            dest:  "/var/lib/jenkins{{ item.path }}{{ item.name }}"
+            owner: "jenkins"
+            group: "jenkins"
+          with_items:
+            - { path: "/jobs/Build-deploy-and-test/", name: "config.xml" }
+            - { path: "/", name: "github-plugin-configuration.xml" }
+            - { path: "/", name: "jenkins.model.JenkinsLocationConfiguration.xml" }
+
+        - name: Restart jenkins service
+          service:
+            enabled: yes
+            name:    jenkins
```

Comment:
You're not just restarting it. You're also enabling it. Please include that information in task name.

Reply:
It is enabled by default after installation, so there is not change here.

Reply:
If you're not changing it, it would be clearer to just not include this option at all.